src/decorator/decorators.ts

```diff
@@ -53,6 +53,7 @@ export function Validate(constraintClass: Function, constraintsOrValidationOptio
  */
 export function ValidateNested(validationOptions?: ValidationOptions) {
     return function (object: Object, propertyName: string) {
+        IsDefined()(object, propertyName);
```

Comment:
Yes for sure, suppose calling `ValidateNested()` on an `undefined` value. Until now, it threw an error. Now it is silent in the case of undefined, but then, we should be informed about the missing property. So basically, for each `ValidateNested()` I add a default `IsDefined()`. From now, if you perform a `ValidateNested()` on an undefined value, you'll get an error "should be defined" when calling `validate()`